applications/nrf_desktop/README.rst

```diff
@@ -153,7 +153,7 @@ Configuration sources
 
 The nRF Desktop project uses the following files as sources of its configuration:
 
-    * DTS files -- reflect the hardware configuration (see :ref:`device-tree`).
+    * DTS files -- reflect the hardware configuration (see :ref:`devicetree`).
```

Comment:
```suggestion
    * DTS files -- reflect the hardware configuration (see :ref:`zephyr:devicetree`).
```
Not a must-fix, but we should be as specific as possible when linking to avoid confusion.

Reply:
CI is green so I will merge the PR and fix this comment in one of our documentation PRs